Cargo.toml

```diff
@@ -136,8 +136,8 @@ webpki-roots = "0.25"
 webrender = { git = "https://github.com/servo/webrender", branch = "0.64", features = ["capture"] }
 webrender_api = { git = "https://github.com/servo/webrender", branch = "0.64" }
 webrender_traits = { path = "components/shared/webrender" }
-wgpu-core = { git = "https://github.com/gfx-rs/wgpu", rev = "f25e07b984ab391628d9568296d5970981d79d8b" }
-wgpu-types = { git = "https://github.com/gfx-rs/wgpu", rev = "f25e07b984ab391628d9568296d5970981d79d8b" }
+wgc = { package = "wgpu-core", git = "https://github.com/gfx-rs/wgpu", rev = "f25e07b984ab391628d9568296d5970981d79d8b" }
+wgt = { package = "wgpu-types", git = "https://github.com/gfx-rs/wgpu", rev = "f25e07b984ab391628d9568296d5970981d79d8b" }
```

Comment:
I think this rename actually makes it harder to understand the dependencies. Let's just stick with the real package names.

Reply:
The main reason for the rename is that code bases already uses wgc and wgt names (ex: [firefox](https://searchfox.org/mozilla-central/rev/e4a222169f9f9b1d8f2f49e4066585b6b9c4dc58/gfx/wgpu_bindings/Cargo.toml#17), [wgpu](https://github.com/gfx-rs/wgpu/blob/9f34acd5671987558038ba0abeb24c69220bd581/Cargo.toml#L54), [wgpu-native](https://github.com/gfx-rs/wgpu-native/blob/a1ede115d0d669b88b34447e58752335cb882e61/Cargo.toml#L24) and servo a little) and that's way it makes sense to follow it (given that hacking servo's webgpu usually requires looking at wgpu/firefox code) also it's less writing and we avoid dealing with `-_` problem.

Reply:
This is a good argument, but let's stick with the full names and only rename when there is a naming conflict. We have a lot more Rust code than Firefox does with a *lot* of dependencies (a number that would be good to reduce). Any kind of obfuscation makes it harder to folks to understand what is going on when they are reading the code for the first time.

Reply:
> We have a lot more Rust code than Firefox does with a _lot_ of dependencies (a number that would be good to reduce). Any kind of obfuscation makes it harder to folks to understand what is going on when they are reading the code for the first time.

wgpu-types and wgpu-core are only used by webgpu crate that has only few direct dependencies, although they are pub reexport via wgt, wgc names. I will remove this from this PR, but I would still like to push it forward in the future (will create issue or maybe zulip channel for more wide discussion). Coincidentally, this remands me of [Wadler's Law](https://wiki.haskell.org/Wadler's_Law).

Ironically, the rename of `mozjs` to `js` bugs me (due to using `mozjs` in other projects), so similar story with reversed roles.

Reply:
In general, using abbreviations for the names of things leads to a very tiny benefit for the person writing the abbreviation, but has a continual cost for everyone trying to understand and implement bug fixes in the code in the future. I don't think it's worth it.  I typically use the rule of thumb established by the [Google C++ guidelines](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules):

> Use names that describe the purpose or intent of the object. Do not worry about saving horizontal space as it is far more important to make your code immediately understandable by a new reader. Minimize the use of abbreviations that would likely be unknown to someone outside your project (especially acronyms and initialisms). Do not abbreviate by deleting letters within a word. As a rule of thumb, an abbreviation is probably OK if it's listed in Wikipedia. Generally speaking, descriptiveness should be proportional to the name's scope of visibility. For example, n may be a fine name within a 5-line function, but within the scope of a class, it's likely too vague.

This is particularly important in a web engine, because the codebase is so big.